### PR TITLE
[WIP][ v7r1] first steps for json serialization

### PR DESCRIPTION
**Pull request by chaen**
BEGINRELEASENOTES
Thank you for writing the text to appear in the release notes. It will show up
exactly as it appears between the two bold lines

Please follow the template:
*Subsystem
NEW/CHANGE/FIX: explanation

For examples look into release.notes

ENDRELEASENOTES

### CODE REVIEW

File: Core/DISET/RequestHandler.py [hunk 1]
```diff
@@ -158,7 +158,9 @@ def __doFileTransfer(self, sDirection):
     if not retVal['OK']:
       raise RequestHandler.ConnectionError("Error while receiving file description %s %s" %
                                            (self.srv_getFormattedRemoteCredentials(), retVal['Message']))
-    fileInfo = retVal['Value']
+
+    # Reconvert to tuple
+    fileInfo = tuple(retVal['Value'])
```
Comment (fstagni) [inline Core/DISET/RequestHandler.py]: why?

File: Core/DISET/TransferClient.py [hunk 2]
```diff
@@ -31,7 +31,8 @@ def _sendTransferHeader( self, actionName, fileInfo ):
       retVal = self._proposeAction( transport, ( "FileTransfer", actionName ) )
       if not retVal[ 'OK' ]:
         return retVal
-      retVal = transport.sendData( S_OK( fileInfo ) )
+      # We need to convert to list
+      retVal = transport.sendData( S_OK( list(fileInfo) ) )
```
Comment (fstagni) [inline Core/DISET/TransferClient.py]: format (here and later)

File: Core/DISET/private/InnerRPCClient.py [hunk 3]
```diff
@@ -58,7 +58,10 @@ def executeRPC( self, functionName, args ):
             return retVal
 
       # Send the arguments to the function
-      retVal = transport.sendData( S_OK( args ) )
+      # Note: we need to convert the arguments to list
+      # We do not need to deseralize it because variadic functions
```
Comment (fstagni) [inline Core/DISET/private/InnerRPCClient.py]: what's `variadic`?

Comment (chaen) [reply Core/DISET/private/InnerRPCClient.py]: `*args, **kwargs` 
but do not bother to review yet, it's far from complete and will be reshuffled

**Comment by chaen** [comment]
Replaced by https://github.com/DIRACGrid/DIRAC/pull/4362

**Pull request closed without merging**